packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1056,6 +1056,108 @@ Expected response body:
 }
 ```
 
+### Azure_ResourceManager_Models_Resources_OperationStatuses_getRgStatus
+
+- Endpoint: `get https://management.azure.com`
+
+  Resource GET operation.
+  Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.Resources/locations/westus/operationStatuses/00000000-0000-0000-0000-000000000000
+  Expected query parameter: api-version=2023-12-01-preview
+
+  Expected response body:
+
+  ```json
+  {
+    "name": "ResourceGroupOperation",
+    "startTime": <any date>,
+    "endTime": <any date>,
+    "percentComplete": 2,
+    "status": "Failed",
+    "ManagementError": {
+      "code": "404",
+      "message": "The error message of status test. ",
+      "target": "The target of status test.",
+      "details": [{
+        "code": "404",
+        "message": "The error message of status test. ",
+        "target": "The target of status test.",
+      }],
+      "additionalInfo": [{
+        "type": "additionalType",
+        "info": "additionalInfo"
+      }]
+    }
+  }
+  ```
+
+### Azure_ResourceManager_Models_Resources_OperationStatuses_getStatus
+
+- Endpoint: `get https://management.azure.com`
+
+  Resource GET operation.
+  Expected path: /providers/Azure.ResourceManager.Models.Resources/locations/westus/operationStatuses/00000000-0000-0000-0000-000000000000
+  Expected query parameter: api-version=2023-12-01-preview
+
+  Expected response body:
+
+  ```json
+  {
+    "name": "TenantOperation",
+    "startTime": <any date>,
+    "endTime": <any date>,
+    "percentComplete": 2,
+    "status": "Failed",
+    "ManagementError": {
+      "code": "404",
+      "message": "The error message of status test. ",
+      "target": "The target of status test.",
+      "details": [{
+        "code": "404",
+        "message": "The error message of status test. ",
+        "target": "The target of status test.",
+      }],
+      "additionalInfo": [{
+        "type": "additionalType",
+        "info": "additionalInfo"
+      }]
+    }
+  }
+  ```
+
+### Azure_ResourceManager_Models_Resources_OperationStatuses_getSubscriptionStatus
```

Comment:
maybe `Azure_ResourceManager_Models_Resources_OperationStatuses_subscription`

Reply:
@iscai-msft 
According to your idea, should `Azure_ResourceManager_Models_Resources_OperationStatuses_getStatus` and `Azure_ResourceManager_Models_Resources_OperationStatuses_getRgStatus` also be modified to `Azure_ResourceManager_Models_Resources_OperationStatuses_tenant` and `Azure_ResourceManager_Models_Resources_OperationStatuses_resourceGroup`?

Reply:
@XiaofeiCao 
> maybe `Azure_ResourceManager_Models_Resources_OperationStatuses_subscription`

Do you have any good ideas for this proposal?

Reply:
@v-hongli1 If you want to add getting operation status, I prefer to add it under each resource type, as we already have them, e.g. TopLevelTrackedResource, NestedChildResource. And future resource types as well, e.g. ExtensionResource, LocationResource, etc.

Reply:
Fixed by commit https://github.com/Azure/cadl-ranch/commit/4e639a8c2464febccec886fcfb3b531b02c0cfb9